dataactbroker/fileRoutes.py

```diff
@@ -376,6 +387,11 @@ def certify_submission(submission):
         if submission.publish_status_id == PUBLISH_STATUS_DICT['published']:
             return JsonResponse.error(ValueError("Submission has already been certified"), StatusCode.CLIENT_ERROR)
 
+        window = get_gtas_window()
+        if not window:
```

Comment:
wait, this logic doesn't make sense. If it isn't the window it can't be certified? It should be `if window:`